vnext/RNW.targets

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
See comments on RNW.props

Reply:
move it to propertysheets\bundle.targets